content/en/blog/2024/prom-and-otel/index.md

```diff
@@ -0,0 +1,564 @@
+scrape your services, and supports the [full set](https://github.com/prometheus/prometheus/blob/v2.28.1/docs/configuration/configuration.md#scrape_config) of configurations in 
+`scrape_config`. 
+
+If you are interested in [exemplars](https://opentelemetry.io/docs/specs/otel/metrics/data-model/#exemplars), which is a recorded value that associates OTel context to a metric 
+event, you can also use the Prometheus receiver to ingest them in the Prometheus 
+format and convert it to OTLP format. This enables you to correlate traces with 
+metrics. 
+
+Something to consider with this component is that it is under active 
+development; as such, it has several [limitations](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/prometheusreceiver/README.md#%EF%B8%8F-warning), including that it’s a stateful component. Additionally, it is recommended to not 
+use this component when multiple replicas of the collector are run, because in 
+this state: 
+* The collector is unable to auto-scale the scraping 
+* If the replicas are running with the same config, it will scrape the targets 
+multiple times
+* You will need to configure each replica with a different scraping config if 
+you want to manually shard the scraping
+
+The [Prometheus exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/prometheusexporter#prometheus-exporter) allows you to ship data in the 
+Prometheus format, which can then be scraped by a Prometheus server. It is used 
+to report metrics via the Prometheus scrape HTTP endpoint. You can learn more by 
+trying out this [example](https://github.com/open-telemetry/opentelemetry-go/tree/main/example/prometheus). 
+
+On the flip side, if you are generating OTel metrics and you want to ship them 
+to a backend that is compatible with Prometheus remote write (such as 
+[Thanos](https://thanos.io/)), you can use the [Prometheus remote write exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/prometheusremotewriteexporter/README.md). Prometheus remote write allows you to write data to external storage – in 
+this case, backends that are compatible with Prometheus. 
+
+## Using the Target Allocator 
+A common challenge with Prometheus is that of scalability, which is the ability 
+to effectively maintain performance and resource allocation while managing an 
+increasing number of monitored targets and metrics. One option to help with this 
+is **sharding** the workload based on labels or dimensions, [which means using multiple Prometheus instances to handle your metrics according to specific parameters](https://www.atatus.com/blog/prometheus-architecture-scalability/#:~:text=Key%20Limitations%20of%20Prometheus,-Long%2DTerm%20Storage&text=Scalability%3A%20Extremely%20big%20environments%20with,availability%2C%20which%20influences%20system%20resilience.). This could help 
```

Comment:
Is the period at the end of the URL superfluous?
```suggestion
is **sharding** the workload based on labels or dimensions, [which means using multiple Prometheus instances to handle your metrics according to specific parameters](https://www.atatus.com/blog/prometheus-architecture-scalability/#:~:text=Key%20Limitations%20of%20Prometheus,-Long%2DTerm%20Storage&text=Scalability%3A%20Extremely%20big%20environments%20with,availability%2C%20which%20influences%20system%20resilience). This could help 
```